layouts/_default/_markup/render-image.html

```diff
@@ -1 +1,21 @@
-<img loading="lazy" src="{{ .Destination | safeURL }}" alt="{{ .Text }}" {{ with .Title}} title="{{ . }}" {{ end }} />
+{{ $destination := .Destination }}
+{{- if not (strings.HasPrefix .Destination "http") -}}
+    {{- $image := (.Page.Resources.ByType "image").GetMatch .Destination -}}
+    {{- if $image }}
+        {{- with .Page -}}
+            {{- $imageOptions := "" -}}
+            {{- with .Params.convertimagesto -}}
+                {{- $imageOptions = printf "%dx%d %s" $image.Width $image.Height . -}}
+            {{- else -}}
+                {{- with .Site.Params.convertimagesto -}}
```

Comment:
Are you sure `.Site.Params` can be called within render hooks?

Reply:
OMG! Yes.

Reply:
LOL! I feel better now 👍 

Thank you for looking at this. I'll look at your changes shortly (for my understanding; it's all about collaboration and learning together, right?)

Reply:
Also note this is actually .Page.Site.Params (because of the with). Plain .Site.Params does not work.